src/platforms/common/performance/index.mdx

```diff
@@ -94,11 +94,11 @@ Test out tracing by starting and finishing a transaction, which you _must_ do so
 
 While you're testing, set <PlatformIdentifier name="traces-sample-rate" /> to `1.0`, as that ensures that every transaction will be sent to Sentry.
 
-Once testing is complete, **we recommend lowering this value in production** by either lowering your <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to dynamically sample and filter your transactions.
+Once testing is complete, you may want to consider setting a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions, based on contextual data. 
 
 <PlatformSection supported={["javascript"]} notSupported={["react-native"]}>
 
-Leaving the sample rate at `1.0` means that automatic instrumentation will send a transaction each time a user loads any page or navigates anywhere in your app, which is a lot of transactions. Sampling enables you to collect representative data without overwhelming either your system or your Sentry transaction quota.
+Leaving the sample rate at `1.0` means that automatic instrumentation will send a transaction each time a user loads any page or navigates anywhere in your app, which can be a lot of transactions. Sampling enables you to collect representative data without overwhelming either your system or your Sentry transaction quota. For lower traffic applications there may not be a concern at all. It is mainly of interest to high load backend applications, where you may want to explore sampling consider load on the network or system resources are high.
```

Comment:
```suggestion
If you leave your sample rate at `1.0`, a transaction will be sent every time a user loads a page or navigates within your app. Depending on the amount of traffic your application gets, this may mean a lot of transactions. If you have a high-load, backend application, you may want to consider setting a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions, based on contextual data.
```
I recommend consolidating the above paragraph with this one into one.